cli/destroy.go

```diff
@@ -10,7 +10,7 @@ import (
 
 var (
 	destroyCmd = &cobra.Command{
-		Use:   "destroy <group|slice...>",
+		Use:   "destroy <group[@slice]...>",
```

Comment:
its the separator as discussed in https://github.com/giantswarm/inago/issues/149